open-telemetry/otel-collector.mdx

```diff
@@ -0,0 +1,133 @@
+
+<Accordion title="Step 1: Install DockerHub or ghcr.io">
+    [Download](https://opentelemetry.io/docs/collector/installation/) the opentelemetry-collector with the same container registry you used to install the MW Agent.
+</Accordion>
+
+<Accordion title="Step 2: Use otelcol-contrib">
+    Ensure the container image refers to the `opentelemetry-collector-contrib` and not core `opentelemetry-collector`.
+</Accordion>
+
+<Accordion title="Step 3: Follow OTel Config">
+    Once you have installed Docker, proceed to the [Configuration Section](/open-telemetry/otel-collector#configuration).
+</Accordion>
+
+# Configuration
+
+<Accordion title="Step 1: Configure Collector">
+    Follow the [OTel Configuration Instructions](https://opentelemetry.io/docs/collector/configuration/) to configure the `opentelemetry-collector-contrib`
+
+    <Note> You must fully configure the OTel Collector before adding more code to the `processors`, `exporters` and `service` section. </Note>
+</Accordion>
+
+<Accordion title="Step 2: Add to Processors">
+
+    Add the following code to the `processors` section:
+
+    ```YAML YAML
+    processors:
+    resourcedetection:
+        detectors: [ env, system ]
+        timeout: 2s
+        system:
+        hostname_sources: [ os ]
+    resource/middleware:
+        attributes:
+        - action: upsert
+        key: mw.account_key
+        value: "<your-mw-api-key>"
+        - action: upsert
+        from_attribute: host.name
+        key: host.id
+    ```
+</Accordion>
+
+<Accordion title="Step 3: Add to Exporters">
+
+    Add the following code to the `exporters` section:
+
+    ```YAML YAML
+    exporters:
+    otlp/middleware:
+        endpoint: "your-endpoint.middleware.io:443"
+        headers:
+        authorization: "<your-mw-api-key>"
+        sending_queue:
+        enabled: true
+        num_consumers: 100
+        queue_size: 10000
+    ```
+</Accordion>
+
+<Accordion title="Step 4: Add to Service">
+    Add the following code within the pipelines section:
+
+    ```YAML YAML
+    service:
+    pipelines:
+        traces:
+        receivers: [ <your-receivers-list-for-traces> ]
+        processors: [ resourcedetection, resource/middleware, 
+                        <your-processor-list-for-traces> ]
+        exporters: [ otlp/middleware, <your-exporters-list-for-traces> ]
+        metrics:
+        receivers: [ <your-receivers-list-for-metrics> ]
+        processors: [ resourcedetection, resource/middleware ]
+        exporters: [ otlp/middleware, <your-exporters-list-for-metrics> ]
+        logs
+        receivers: [ <your-receivers-list-for-logs> ]
+        processors: [ resourcedetection, resource/middleware, 
+    <your-processors-list-for-logs> ]
+        exporters: [otlp/middleware, <your-exporters-list-for-logs>]
+    ```
+</Accordion>
+
+# Limitations
+
+Metrics such as bandwidth and disk speed are not currently collected by the OpenTelemetry Collector and will contain `0` in those fields.
```

Comment:
Recommend an additional sentence that calls out where this will be impacted. E.g. "This will impact the infrastructure pages as well as any custom dashboards built with these metrics."